src/jit/inline.cpp

```diff
@@ -153,6 +153,172 @@ const char* inlGetImpactString(InlineObservation obs)
 }
 
 //------------------------------------------------------------------------
+// inlGetDecisionForRuntime: translate decision into a CorInfoInline
+//
+// Arguments:
+//    d - the decision in question
+//
+// Return Value:
+//    CorInfoInline value representing the decision
+
+CorInfoInline inlGetDecisionForRuntime(InlineDecision d)
```

Comment:
I have more edits coming in this file, so unless I need to make other changes for this PR, I will fix this subsequently.